.changeset/curvy-bulldogs-boil.md

```diff
@@ -0,0 +1,11 @@
+---
+"@comet/admin": minor
+---
+
+Add new custom `DialogTitle` to @comet/admin
+
+-   optional `onClose` prop adds Close Button to the Component
+
+Example:
+
+-   `<DialogTitle onClose={() => {...}} />`
```

Comment:
Mentioning "@comet/admin" is not necessary as this changelog is explicitly for "@comet/admin"